File: youqugou-interface/src/main/java/cn/itcast/core/service/BrandAuditService.java
package cn.itcast.core.service;


import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.good.Brand;

/**
 * 贾运通2018/12/30
 */
public interface BrandAuditService {
    PageResult searchBrandAudit(Integer pageNum, Integer pageSize, Brand brand);

    void updateStatus(Long[] ids, String status);
}

File: youqugou-web-manager/src/main/webapp/js/service/brandAuditService.js
/**
 * 贾运通2018/12/30
 */

app.service("brandAuditService", function ($http) {
    this.searchBrandAudit = function (currentPage, itemsPerPage,searchEntity) {
        return $http.post("../brandAudit/searchBrandAudit.do?pageNum=" + currentPage
            + "&pageSize=" + itemsPerPage, searchEntity);
    }


    this.updateStatus = function (ids,status) {
        return $http.get('../brandAudit/updateStatus.do?ids='+ids+"&status="+status);
    }


});